app/app.rb

```diff
@@ -0,0 +1,81 @@
+require 'sinatra/base'
```

Comment:
WIth regards to your question about whether your routes are RESTful, I've commented on the specific routes I think could be changed to make them more restful.

For non-session routes, I wonder if you've seen [this](https://github.com/sjmog/rest)? It's a document and game you can use to check your REST understanding. [This](https://gist.github.com/alexpchin/09939db6f81d654af06b) is also a code snippet where someone summarises their restful routes.
For the _session_ routes, I found [this guide](https://stackoverflow.com/a/7252311)